docs/src/prototypes/MenuButton/MenuButton.tsx

```diff
@@ -150,10 +150,11 @@ class MenuButton extends React.Component<MenuButtonProps, MenuButtonState> {
     const { button, disabled, menu, placement } = this.props
     const { menuOpen } = this.state
     const [position, align] = _.split(placement, '-') as [Position, Alignment]
-    const accessibilityBehavior: ReactAccessibilityBehavior = menuButtonBehavior({
-      ...this.props,
-      ...this.state,
-    })
+    const accessibilityBehavior: ReactAccessibilityBehavior = {
+      attributes: {},
```

Comment:
why do you an empty `attributes` object here? I see attributes already exist in `menuButtonBehavior`?

Reply:
Extracted this to empty behavior